### PR TITLE
Editorial updates

### PR DESCRIPTION
**Pull request by SirSpidey**
- Update external link syntax
- Add conrefs
- Remove refs to Bluemix (including in token URL)

### CODE REVIEW

File: migrate.md [hunk 1]
```diff
@@ -41,16 +41,16 @@ If you created service instances before November 2018, then depending on the loc
 ## Migrating a service instance
 {: #migrate-task}
 
-If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-migrate){: new_window}.
+If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation](/docs/resources?topic=resources-migrate){: external}.
 
-Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more information, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: new_window}.
+Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more details, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: external}.
```
Comment (jeffpk62) [inline migrate.md]: Oh, my, changing "for more information" to "for more details"?  Unbelievable.

File: migrate.md [hunk 2]
```diff
@@ -41,16 +41,16 @@ If you created service instances before November 2018, then depending on the loc
...
 {: note}
 
 To migrate your service instance, complete these steps:
 
 1.  Determine which resource group you want to move the service instance to.
 
-    See [Best practices for organizing resource in a resource group ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-bp_resourcegroups) for tips.
+    See [Best practices for organizing resource in a resource group](/docs/resources?topic=resources-bp_resourcegroups) for tips.
```
Comment (jeffpk62) [inline migrate.md]: Should this have an `external` on it, Allen?  Same with a few that follow.

File: migrate.md (hunk 1 above)
Comment (SirSpidey) [reply migrate.md]: Not sure where that came from. Perhaps it's a regression. Will reverse.

File: migrate.md (hunk 2 above)
Comment (SirSpidey) [reply migrate.md]: Nope. It's in the doc app.

**Comment by jeffpk62** [comment]
@SirSpidey You should take a look at the comments I entered, to be certain they're what you intend.

File: migrate.md (hunk 2 above)
Comment (SirSpidey) [reply migrate.md]: But i see another one does, so I removed that too.

**Pull request merged**